includes/Generator/Plugins/SchemaOrg.php

```diff
@@ -179,9 +186,28 @@ private function getAuthorMetadata() {
 			$sitename = '';
 		}
 
+		$server = MediaWikiServices::getInstance()->getMainConfig()->get( 'Server' );
+
+				try {
+			$logo = MediaWikiServices::getInstance()->getMainConfig()->get( 'Logo' );
+			$logo = wfExpandUrl( $logo );
+			$data['url'] = $logo;
+		  } catch ( Exception $e ) {
+			// Uh oh either there was a ConfigException or there was an error expanding the URL.
+			// We'll bail out.
+			$logo = [];
```

Comment:
Should this be a string instead of an empty array? 
```$logo``` is currently the absolute url to the image